code/api/views.py

```diff
@@ -473,6 +480,32 @@ def post(self, request, author_id, post_id):
 
         return redirect('socialDistribution:single-post', post_type="local", id=post_id)
 
+@method_decorator(csrf_exempt, name='dispatch')
+class PostCommentsSingleView(View):
+    '''
+        HANDLE Singular Comment GET 
+    '''
+
+    def get(self, request, author_id, post_id, comment_id):
+        logger.info(f"GET /author/{author_id}/posts/{post_id}/comments/{comment_id} API endpoint invoked")
+
+        # Send comment
+        try:
+            post = get_object_or_404(LocalPost, id=post_id)
+            author = get_object_or_404(LocalAuthor, id=author_id)
+            # Check if the post author match with author in url
+            if post.author.id != author.id:
+                return HttpResponseNotFound()
+            
+            comment = get_object_or_404(Comment, id=comment_id)
```

Comment:
`get_object_or_404` throws a Http404 error, but then we catch it and return a 500 error instead of 404. Can you modify it to avoid catching the Http404? Thanks

Reply:
also would be good to throw 404 if the comment doesn't correspond to post